Dockerfile

```diff
@@ -0,0 +1,61 @@
+# translate/pootle
+#
+# VERSION	0.0.1
+
+FROM debian
+
+MAINTAINER Ryan Northey <ryan@synca.io>
+
+RUN apt-get update && \
+    apt-get install -y \
+            python-pip \
+	    libxml2-dev \
+	    libxslt-dev \
+	    python-dev \
+	    zlib1g-dev \
+	    build-essential \
+	    redis-server \
+	    sudo && \
+    apt-get clean && \
+    pip install virtualenv && \
+    groupadd -r pootle && \
+    useradd -m -d /home/pootle \
+            -k /etc/skel \
+            -s /bin/bash \
+            -g pootle pootle && \
+    /etc/init.d/redis-server start && \
+    sudo -u pootle bash -c "\
+    	 mkdir ~/dev/env -p && \
+	 cd ~/dev/env && \
+    	 virtualenv . && \
+    	 . bin/activate && \
```

Comment:
the less installed programs the better

if you were to build a dev Dockerfile then i would totally agree